app/src/main/java/com/example/yard/Activities/PostActivity.java

```diff
@@ -167,6 +179,24 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
         Toast.makeText(this, "Picture wasn't taken!", Toast.LENGTH_SHORT).show();
       }
     }
+    if (requestCode == GALLERY_REQUEST_CODE) {
+      if (resultCode == RESULT_OK) {
+        Uri fileUri = data.getData();
+        ivImage.setImageURI(fileUri);
+        photoFile = new File(getPaths(fileUri));
+      }
+    }
+  }
+
+  public String getPaths(Uri uri) {
```

Comment:
Camera uploading is already in another PR. Please make sure that you're excluding them from your font polishing.